2주차/12789/12789_python_토르.py

```diff
@@ -0,0 +1,24 @@
+from collections import deque
+
+N = int(input())
+students = list(map(int, input().split()))
+
+waiting_stack = []
+done = []
+answer = [i for i in range(1, N+1)]
+
+for i in range(len(students)):
```

Comment:
```suggestion
for index, value in enumerate(student):
```
이 상황에서는 enumerate를 사용할 수 있지 않을까요??